app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/MyDetailsOverviewRow.java

```diff
@@ -42,6 +43,14 @@ public void setSummary(String mSummary) {
         this.mSummary = mSummary;
     }
 
+    public void setProgress(int progress) {
```

Comment:
```suggestion
    public void setProgress(@IntRange(from = 0, to = 100) int progress) {
```

`import androidx.annotation.IntRange;`